cockroachdb/src/jepsen/cockroach.clj

```diff
@@ -70,6 +70,10 @@
                   (auto/start! test node)
                   (Thread/sleep 5000)) ; Give it time to join
 
+                (when (= node (jepsen/primary test))
+                  (auto/init! node)
+                  (Thread/sleep 5000))
```

Comment:
ditto

Reply:
Also the code below "restart cluster to work around balancing bug" is not necessary any more. 

(I would rather update this code to instead wait for up-replication, if there's an easy way to add that)

Reply:
How long have these things been fixed? Remember that we run jepsen (from the same branch) against all supported release branches of cockroachdb. 

Even if the sleeps aren't necessary to work around old bugs, we shouldn't completely get rid of them until we're able to add an explicit "wait for upreplication/rebalancing" step.

Reply:
Ah good point. Maybe keep them then.